src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -7798,12 +7798,19 @@ def test_aks_create_with_monitoring_aad_auth_uai_with_syslog(self, resource_grou
         aks_name = self.create_random_name('cliakstest', 16)
         self.create_new_cluster_with_monitoring_aad_auth(resource_group, resource_group_location, aks_name, user_assigned_identity=True, syslog_enabled=True)
 
-    def create_new_cluster_with_monitoring_aad_auth(self, resource_group, resource_group_location, aks_name, user_assigned_identity=False, syslog_enabled=False, data_collection_settings=None):
+    @live_only()
```

Comment:
Why do you need to mark this test as `@live_only()`? Please add some comments to explain the reason

Reply:
> Why do you need to mark this test as `@live_only()`? Please add some comments to explain the reason

done.